src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -555,6 +555,7 @@ public ColumnFamilyStore(Keyspace keyspace,
         }
         writeHandler = new CassandraTableWriteHandler(this);
         streamManager = new CassandraStreamManager(this);
+        // TODO (now): support mock?
```

Comment:
`CassandraTableRepairManager` already carries a `RepairContext`, so it could delegate to that `ActiveRepairService`. What's left...the `CompactionManager`? I'm guessing the side effects of using non-mocked versions of those are benign for now?

Reply:
removed the TODO... I still think the `SharedContext` `TableRepairManager repairManager(ColumnFamilyStore store)`...